docs/contracts.rst

```diff
@@ -976,9 +975,9 @@ contracts (``L.f()`` if ``L`` is the name of the library). Furthermore,
 if the library were a base contract. Of course, calls to internal functions
 use the internal calling convention, which means that all internal types
 can be passed and memory types will be passed by reference and not copied.
-In order to realise this in the EVM, code of internal library functions
-(and all functions called from therein) will be pulled into the calling
-contract and a regular ``JUMP`` call will be used instead of a ``DELEGATECALL``.
+To realize this in the EVM, code of internal library functions
```

Comment:
yeah definitely revealing my American bias but also apparently the 'ise' endings also aren't used in British periodicals etc either.   I'm fine with dropping this change especially since the original writer used British locutions in other places.